keyboards/handwired/marek128b/ergosplit44/keymaps/default/keymap.c

```diff
@@ -0,0 +1,63 @@
+// Copyright 2023 QMK
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include QMK_KEYBOARD_H
+#include "quantum.h"
+
+enum layer_names {
+    _QWERTY,
+    _COLEMAK,
+   
+};
+
+enum custom_keycodes {
+    QWERTY,
+    COLEMAK,
+};
+
```

Comment:
```suggestion
```

Reply:
Please do not mark suggestions as resolved without addressing them

Reply:
sorry but would you please explain why you are suggesting this change as i see this change not to be nessesary and easier to read without your suggestion. please explain if i am right here or if I'm completle wrong

Reply:
The contents of this `enum` are not used anywhere. You have an
```c
enum layer_names {
    _QWERTY,
    _COLEMAK
};
```
which you have used accordingly when assigning layers: `[_COLEMAK] = LAYOUT( ... )`. Meaning keycodes like `TG(_COLEMAK)` will work and whatnot. From this enumeration:
```c
enum custom_keycodes {
    QWERTY,
    COLEMAK
};
```
these elements are **not** used anywhere.